one_fm/www/job_applicant_magic_link/index.py

```diff
@@ -172,13 +177,13 @@ def upload_image():
                 if(k=="given_names" and v):
                     for i, j in enumerate(v):
                         if i==0:
-                            frappe.db.set_value(reference_doctype, reference_docname, 'one_fm_first_name', j)
+                            frappe.db.set_value(reference_doctype, reference_docname, 'one_fm_first_name', j.title())
```

Comment:
Make it title case from the validate method of job applicant too.

Reply:
Sure